src/wms/Database.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import knex, { Knex } from 'knex';
+import dateformat from 'dateformat';
+import StaffSchedule from './Interfaces/StaffSchedule';
+
+export class Database {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private connection?: Knex<any, unknown[]>;
+
+  constructor() {
+    this.connection = knex({
+      client: 'mysql',
+      connection: {
+        host: process.env.WMS_HOST,
+        port: parseInt(process.env.WMS_PORT, 10),
+        user: process.env.WMS_USER,
+        password: process.env.WMS_PASSWORD,
+        database: process.env.WMS_SCHEMA,
+        dateStrings: true,
+      },
+    });
+  }
+
+  public async getstaffSchedules(): Promise<StaffSchedule[]> {
```

Comment:
Do we need to add a try...catch to this method in case the connection to the database isn't successful? Or is this already caught within ExportEvents.ts

Reply:
It should get caught in the try-catch in the handler, but the error was getting swallowed in ExportEvents.ts. Just made a change so this is not so.